src/MissionManager/SurveyMissionItem.cc

```diff
@@ -1062,7 +1062,7 @@ bool SurveyMissionItem::_appendMissionItemsWorker(QList<MissionItem*>& items, QO
         MissionItem* item = new MissionItem(seqNum++,
```

Comment:
@DonLakeFlyer We never seem to hit this point after flying the Survey (even if there is no refly). Some sort of a logical issue somewhere. Can you have a look?